<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MindKitForm</class>
 <widget class="QWidget" name="MindKitForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetLeft" native="true">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>675</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetLeft
{
	border-image: url(:/resource/images/common/Transparent_White_90.png);
}</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>161</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/bb_mind_kit/kit_Arrow.png);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>在右边找到你想加入的模块拖拽将其加入你的作品当中</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widgetRight" native="true">
     <property name="minimumSize">
      <size>
       <width>845</width>
       <height>675</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetRight
{
	border-image: url(:/resource/images/common/Transparent_Black_90.png);
}</string>
     </property>
     <widget class="QPushButton" name="pushButtonReturn">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>0</y>
        <width>48</width>
        <height>39</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/common/Back.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MindKitPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>131</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/bb_mind_kit/kit_Button.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MindKitPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/bb_mind_kit/kit_RotationSensor.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MindKitPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>101</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/bb_mind_kit/kit_LightSensor.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MindKitPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>210</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/bb_mind_kit/kit_CloseSensor.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MindKitPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/bb_mind_kit/kit_LED.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MindKitPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>430</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/bb_mind_kit/kit_Buzzer.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MindKitPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>260</y>
        <width>171</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/bb_mind_kit/kit_Servo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MindKitPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>490</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/bb_mind_kit/kit_IRRec.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MindKitPushButton</class>
   <extends>QPushButton</extends>
   <header>MindKitPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
